akka-http-testkit/src/main/resources/application.conf

```diff
@@ -0,0 +1 @@
+akka.http.testkit.marshalling.timeout = 1
```

Comment:
This one should be `reference.conf` rather than `application.conf` as it is a default value and then users will use their `application.conf` to override it if they wish. 

The hoconf file format allows for actual time with units, so that isn't implicit, so if you specify this as "1 s" instead. Here's an example how to read a duration field: https://github.com/akka/akka-http/blob/1d1bfa3f796f3d008676c14e10bcd8daff42dd81/akka-http-tests/src/multi-jvm/scala/akka/http/AkkaHttpServerLatencyMultiNodeSpec.scala#L154